linux_os/guide/system/accounts/accounts-banners/banner_etc_motd/tests/banner_etc_motd_ospp_usbcg_banner.pass.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp
```

Comment:
Are you sure about the profile here? `xccdf_org.ssgproject.content_profile_ospp` is not STIG, except that our RHEL8 STIG draft is based on it.